### ISSUE TITLE
Theming: Bootstrap rules not applied

### ISSUE THREAD

**Issue by EricSch** [telerik/kendo-angular#965]
* Bug report 

## Current behavior
DropDowns / DateInputs aren't taking the whole width, in my case col-md-6 (50%)

## Expected behavior
Controls should fit the defined bootstrap width

## Minimal reproduction of the problem with instructions
http://plnkr.co/edit/GeNsU2wJLV8RYqrUFrNa?p=preview

## Environment

Package versions:
    "@progress/kendo-angular-buttons": "1.1.0",
    "@progress/kendo-angular-dateinputs": "1.2.0",
    "@progress/kendo-angular-dialog": "1.1.0",
    "@progress/kendo-angular-dropdowns": "1.2.1",
    "@progress/kendo-angular-excel-export": "1.0.4",
    "@progress/kendo-angular-grid": "1.4.2",
    "@progress/kendo-angular-inputs": "1.3.0",
    "@progress/kendo-angular-intl": "1.2.2",
    "@progress/kendo-angular-l10n": "1.0.3",
    "@progress/kendo-angular-popup": "1.2.1",
    "@progress/kendo-angular-sortable": "1.0.3",
    "@progress/kendo-angular-upload": "1.2.1",
    "@progress/kendo-data-query": "1.0.7",
    "@progress/kendo-date-math": "1.0.3",
    "@progress/kendo-drawing": "1.4.0",
    "@progress/kendo-file-saver": "1.0.4",
    "@progress/kendo-theme-bootstrap": "2.8.1",
    "@telerik/kendo-intl": "1.2.1"

Browser:
- Chrome (desktop) version 61

**Comment by tsvetomir** [comment]
Looks like a duplicate of telerik/kendo-angular#259

**Comment by EricSch** [comment]
Ok, and I think, you should change your bootstrap theme. In Bootstrap the controls have no default width, they are using all available space.

**Comment by gyoshev** [comment]
@EricSch the inputs have a 100% width, because the Bootstrap input behavior is the same. We even have a suggestion for adding this to other themes (#721). The default width is rather arbitrary, [hardcoded at `12.4em`](https://github.com/telerik/kendo-theme-default/blob/develop/scss/input/_layout.scss#L18). I can't remember why this is the case, even in Kendo UI for jQuery.

**Comment by EricSch** [comment]
@gyoshev So are you saying, the kendo-theme-default overwrites the bootstraps width? 
I never looked deep into your styles, but I did see, that the kendo-theme is also always included. Is your bootstrap theme depending on the kendo-theme-default?

**Comment by gyoshev** [comment]
Yes, the Bootstrap and Material themes depend on the Default theme for their layout styles. Only the Default theme introduces the odd `12.4em` size for inputs, and we consider removing it. For comparison, the Bootstrap framework has defined `.form-control { display: block; width: 100%; }`, so every input is always as wide as its container. This results in better styles by default, as all form elements are aligned with their container.

**Comment by EricSch** [comment]
@gyoshev Thank you, yes, this i was expecting too. Any timeline?

**Comment by gyoshev** [comment]
Since the change may break existing applications, it is appropriate to land in version 3.0 of themes. It may take a while; you can subscribe to #721 to get a notification when it is merged.

As a workaround, you can override this with `$textbox-default-width: 100%`, as shown in the [styling help topic](https://www.telerik.com/kendo-angular-ui/components/styling/#toc-use-the-build-process-of-the-application).
